src/webui/www/private/confirmdeletion.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="UTF-8" />
-    <title>QBT_TR(Deletion confirmation - qBittorrent)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <title>QBT_TR(Remove confirmation - qBittorrent)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
```

Comment:
Maybe `Removal` fits better?

Reply:
I rephrased it to be more specific. This should be more similar to what Windows dialog titles uses: they refer to the specific action in the title: https://www.partitionwizard.com/images/uploads/articles/2021/02/win10-confirm-delete/win10-confirm-delete-1.png